tool/plugin/lib/globals.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 The Chromium Authors. All rights reserved. Use of this source
+// code is governed by a BSD-style license that can be found in the LICENSE file.
+
+const Map<String, String> plugins = const {
```

Comment:
This could use a more descriptive name; `plugins` ==> `supportedPlatformVersions'? `currentIntelliJVersions`?

Reply:
True, I'm not sure why we used that name. I'll do something in the next PR;  wanted this one to be as straight-forward as possible.